thunderhub.patch

```diff
@@ -9,15 +9,15 @@
  # ---------------
  # Install Dependencies
  # ---------------
--FROM node:14.15-alpine as deps
-+FROM arm64v8/node:14.15-alpine as deps
+-FROM node:16.15.1-alpine as deps
++FROM arm64v8/node:16.15.1-alpine as deps
  
  WORKDIR /app
  
  # Install dependencies neccesary for node-gyp on node alpine
  RUN apk add --update --no-cache \
    libc6-compat \
-   python \
+   python3 \
```

Comment:
I assume this is because TH has switched to python3 under the hood?

Reply:
same as above